README.md

```diff
@@ -1,55 +1,118 @@
 
 ## Launch a local setup including a Relay Chain and a Parachain
 
 Quick: see [polkadot-launch](https://github.com/paritytech/polkadot-launch.git)
 ```
 node ../polkadot-launch/dist/cli.js launch-kusama-local-with-shell.json
 ```
+or
+```
+polkadot-launch launch-kusama-local-with-shell.json
+```
+depending on how you installed `polkadot-launch`.
 
+### Manually Launch a local Rococo Testnet
 
-### Manually Launch Local Rococo Testnet
+#### Launch a Rococo Relay Chain
 
+First, check out which polkadot release to use in the .json files in the [polkadot-launch](./polkadot-launch) folder:
+```json
+{
+	"relaychain": {
+		"bin": "../../../bin/polkadot-0.9.18", // <-- release to use
+    		"chain": "kusama-local",
+        // --snip--
+```
+Then get and build `polkadot`:
 ```bash
 # Compile Polkadot with the real overseer feature
 git clone https://github.com/paritytech/polkadot
-git checkout <release tag>  // release tag that matches the branch id in the polkadot-deps
+# Switch into the Polkadot directory
+cd polkadot
+# Checkout the proper commit
+git checkout <release tag>  # release tag, just like in the example above: polkadot-0.9.18
 cargo build --release
+# If this fails due to  `feature `edition2021` is required` you have set an outdated rust default version. To fix this, you can simply run:
+cargo +nightly-2022-01-31 build --release # or any other up-to-date nightly version
 
 # Generate a raw chain spec
 ./target/release/polkadot build-spec --chain rococo-local --disable-default-bootnode --raw > rococo-local-cfde.json
+```
+Start the `Alice` validator:
+```bash
+./target/release/polkadot --chain rococo-local-cfde.json --alice --validator --tmp --port 30333 --ws-port 9944
+```
+When the node starts you will see several log messages. Take note of the node's Peer ID in the logs. We will need it when connecting other nodes to it. It will look something like this:
+```bash
+Local node identity is: 12D3KooWGjsmVmZCM1jPtVNp6hRbbkGBK3LADYNniJAKJ19NUYiq
+```
+Start the `Bob` validator (in a second terminal):
+```bash
+./target/release/polkadot --chain rococo-local-cfde.json --validator --bob --bootnodes /ip4/<Alice IP>/tcp/30333/p2p/<Alice Peer ID> --tmp --port 30334 --ws-port 9945
+```
+
+More information can be found in the Substrate tutorial [Start a Relay Chain](https://docs.substrate.io/tutorials/v3/cumulus/start-relay/)
+
+#### Reserve a para ID
+Go to [Polkadot Apps](https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9944#/parachains/parathreads). Register the parachain via
+Network > Parachains sub-page, click on Parathreads tab and use the + ParaId button. After registering, the collator should start producing blocks when the next era starts.
+![image](https://d33wubrfki0l68.cloudfront.net/ab3d311e37364a9706f2747b98b24fc259398152/2c4ba/static/4e9213b9ee2f65cc7fa9ccddd73679a3/c1b63/paraid-reserve.png)
 
```

Comment:
This is only relevant in case you use an other option for reserving your parachain slot than the one described in this README.

Citing https://docs.substrate.io/tutorials/v3/cumulus/connect-parachain/#registration-transaction:

> Option 2: slots.forceLease

This is the more formal flow for registration used in production (with the exception of the use of sudo to force a slot lease): you register your reserved paraID with the same account that reserved it, or use sudo with a registrar.forceRegister extrinsic if you wish.

Reply:
OK. Thanks.